hack/cherry-pick.sh

```diff
@@ -71,20 +73,15 @@ if git_status=$(git status --porcelain --untracked=no 2>/dev/null) && [[ -n "${g
   exit 1
 fi
 
-if [[ -e "${REBASEMAGIC}" ]]; then
-  echo "!!! 'git rebase' or 'git am' in progress. Clean up and try again."
+while unmerged=$(git status --porcelain | grep ^U) && [[ -n ${unmerged} ]] || [[ -e "${REBASEMAGIC}" ]]; do
```

Comment:
>Why use while instead of if?

good catch, I'll update to use if

>What doest REBASEMAGIC means or comes from?

Defined on line 34. Its a way to check if a rebase is in process. I'm not sure if our local checkout could ever be in such a state (maybe if someone modified the local checkout on disk directly), but the original code used it to check if the local checkout was in a consistent state and I think its a good idea to continue using it.

Reply:
Oh I searched for it and didn’t find it